eng/bomgenerator/buildAndRun.cmd

```diff
@@ -0,0 +1,6 @@
+setJAVA_HOME=c:\Program Files\AdoptOpenJDK\jdk-11.0.10.9-hotspot
```

Comment:
Type on this line, also we should make a requirement to run this script that Java 11+ is set as the `JAVA_HOME` env var. For the same reasons as the comment below where this isn't easily portable.

Reply:
Removed this. I was testing some changes. We do not really need this. Sorry!